core/lib/via_btc_client/src/inscriber/internal_type.rs

```diff
@@ -0,0 +1,38 @@
+use bitcoin::taproot::ControlBlock;
+use bitcoin::Amount;
+use bitcoin::Transaction;
+use bitcoin::TxIn;
+use bitcoin::TxOut;
+pub use bitcoin::Txid;
+
+pub struct CommitTxInputRes {
+    pub commit_tx_inputs: Vec<TxIn>,
+    pub unlocked_value: Amount,
+    pub inputs_count: u32,
+    pub utxo_amounts: Vec<Amount>,
+}
+
+pub struct CommitTxOutputRes {
+    pub commit_tx_change_output: TxOut,
+    pub commit_tx_tapscript_output: TxOut,
+    pub commit_tx_fee_rate: u64,
+    pub _commit_tx_fee: Amount,
+}
+
+pub struct RevealTxInputRes {
+    pub reveal_tx_input: Vec<TxIn>,
+    pub prev_outs: Vec<TxOut>,
+    pub unlock_value: Amount,
+    pub control_block: ControlBlock,
+}
+
+pub struct RevealTxOutputRes {
```

Comment:
Make sure to have a consistent naming for the struct fields, so the suggestion is to rename `reveal_change_output` to `reveal_tx_change_output` etc.